browser/src/ReadData/ReadData.tsx

```diff
@@ -517,102 +541,126 @@ const ReadDataContainer = ({ datasetId, variantIds }: ReadDataContainerProps) =>
   `
 
   return (
-    <BaseQuery operationName="ReadData" query={query} url="/reads/">
-      {({ data, error, graphQLErrors, loading }: any) => {
-        if (loading) {
-          return <StatusMessage>Loading reads...</StatusMessage>
-        }
-
-        if (error || !data) {
-          return <StatusMessage>Unable to load reads</StatusMessage>
-        }
-
-        const variants = variantIds.map((variantId) => {
-          const [chrom, pos, ref, alt] = variantId.split('-')
-          return { chrom, pos: Number(pos), ref, alt }
-        })
-
-        // Assume all variants are on the same chromosome
-        const { chrom } = variants[0]
+    <>
+      <LoadReadsControls>
+        <Button id="load-once" disabled={loadOnce} onClick={() => setLoadOnce(!loadOnce)}>
+          Load once
+        </Button>
+        <div>
+          <br />
+          <span>
+            <CheckboxInput
+              id="always-load"
+              checked={alwaysLoad}
+              onChange={(e: any) => {
```

Comment:
there should be a type you can use for `e` here, `MouseEvent` or something similar IIRC